src/scancode/cli.py

```diff
@@ -209,6 +209,7 @@ class ScanCommand(utils.BaseCommand):
 formats = ('json', 'html', 'html-app',)
 
 def validate_formats(ctx, param, value):
+    value = value.lower()
```

Comment:
I wonder if the new code should not be more or less like this:
```
def validate_formats(ctx, param, value):
    value_lower = value.lower()
    if value_lower not in formats:
        # render using a user-provided custom format template
        if not os.path.isfile(value):
            raise click.BadParameter('Invalid template file: "%(value)s" does not exists or is not readable.' % locals())
    else:
        value = value_lower
    return value
```

The rationale being that the value could be either a format or a file path... If you lower the path on a case-sensitive OS it may not longer be valid.